src/opustrainer/modifiers/placeholders.py

```diff
@@ -252,18 +253,20 @@ class PlaceholderTagModifier(Modifier):
 
     def __init__(self, probability: float=0.0, custom_detok_src: Optional[str]=None, custom_detok_trg: Optional[str]=None,
         spm_vocab: Optional[Path]=None,
-        template: str="__source__ {src} __target__ {trg} __done__", augment: float=0, replace:float=0):
+        template: str="__source__ {src} __target__ {trg} __done__", augment: float=0, replace:float=0, tag:float=1):
         super().__init__(probability)
 
         self.template = template
+        self.custom_detok_src = custom_detok_src
```

Comment:
it's validated in the next line inside `make_detokenizer` which does the splitting and check that the key is in. a dictionary:
```
        self.src_retokenizer = Retokenizer(
            detokenizer=make_detokenizer(custom_detok_src) ...
```

Reply:
There is some validation, yes, but it's not ergonomic to use it afterwards since you are dealing with raw string manipulation rather than working with a structured class.